install_scripts/templates/replicated-operator.sh

```diff
@@ -206,7 +206,7 @@ write_upstart_services() {
 {% include 'upstart/replicated-operator.conf' %}
 EOF
     cat > /etc/init/replicated-operator-stop.conf <<-EOF
-{% include 'upstart/replicated-ui-stop.conf' %}
+{% include 'upstart/replicated-operator-stop.conf' %}
```

Comment:
this change confuses me
what does it do?

Reply:
idk. its a bug but no one uses upstart anymore anyway